pkgs/development/python-modules/pygtkglext/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, buildPythonPackage, pkgconfig, gtk2, pygtk, gtkglext, pygobject2
+, libglade ? null, libGLU, xorg, isPy3k }:
+
+stdenv.mkDerivation rec {
+  pname = "pygtkglext";
+  version = "1.1.0";
+  name = pname + "-" + version;
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.